waku/node/v2/rpc/wakucallsigs.nim

```diff
@@ -6,6 +6,7 @@ proc waku_publish(topic: string, message: seq[byte]): bool
 proc waku_publish2(topic: string, message: seq[byte]): bool
 proc waku_subscribe(topic: string): bool
 proc waku_query(topics: seq[string]): bool
+proc waku_filter(topic: string, contentTopics: seq[seq[string]]): bool
```

Comment:
Missing something obvious probably, but how come this isn't a seq[string]?

Reply:
If you mean for the topic part, because the FilterRequest only has one topic. It also contains an array of ContentFilters that contain an array of topics, therefore its the array of arrays there.

Reply:
ah right, you can have multiple content filters with multiple content topics? shouldn't the name be contentFilters then?

Reply:
yeah could rename.